packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
 * Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
+* Expose labelType for NavigationRails.
+* Add mapNavigationRailDestination to apply custom Destinations.
```

Comment:
nit: should change this bullet to mention `navigationRailDestinationBuilder` instead.